src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+public class Book {
+    private long id;
+    private String title;
+    private int price;
+
+    private Book(long id, String title, int price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }
+
+    private Book(String title, int price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    public static Book of(long id, String title, int price) {
+        return new Book(id, title, price);
+    }
+
+    public static Book of(String title, int price) {
+        return new Book(title, price);
+    }
```

Comment:
Factory methods 'of' are generally used to create instances without using the 'new' keyword directly. However, these should be used consistently. If you decide to keep the factory methods, you should also ensure that all objects are created through them for consistency.